charts/sophora-schema-docs/templates/deployment.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "sophora-schema-docs.fullname" . }}
+  labels:
+    {{- include "sophora-schema-docs.labels" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+      {{- include "sophora-schema-docs.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      labels:
+        {{- include "sophora-schema-docs.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      containers:
+        - name: {{ .Chart.Name }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          volumeMounts:
+            - mountPath: /app/config
```

Comment:
If you want to load the application configuration, you might want to use a config map. See for example https://github.com/subshell/helm-charts/blob/main/charts/sophora-indexing-service/templates/deployment.yaml#L110C6-L110C6 and https://github.com/subshell/helm-charts/blob/main/charts/sophora-indexing-service/templates/sisi-config.yaml .

```yaml
- name: config
  configMap:
  name: {{ include "sophora-schema-docs.name" . }}
```